From 560ad49715112ee5c32927bf5b2291bf3cd6e269 Mon Sep 17 00:00:00 2001
From: Antal Nemes <antal.nemes.hu@gmail.com>
Date: Thu, 1 Oct 2020 15:04:17 +0200
Subject: [PATCH] logwriter: destination timeout to verbose from notice

After this change, these two logs will occur together:

```
Destination timeout has elapsed, closing connection; fd='12'
Destination timed out, reaping; template='/dev/stdout', filename='/dev/stdout'
```

Originally, only the first one was visibe (without verbose
flag). Withouth the context of the second, this can cause confusion.

Signed-off-by: Antal Nemes <antal.nemes.hu@gmail.com>
Bug-Debian: https://bugs.debian.org/996246
Bug: https://github.com/syslog-ng/syslog-ng/issues/3506
Reviewd-By: Axel Beckert <abe@debian.org>
---
 lib/logwriter.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/lib/logwriter.c
+++ b/lib/logwriter.c
@@ -1315,8 +1315,8 @@
   LogWriter *self = (LogWriter *) cookie;
 
   g_assert(!self->io_job.working);
-  msg_notice("Destination timeout has elapsed, closing connection",
-             evt_tag_int("fd", log_proto_client_get_fd(self->proto)));
+  msg_verbose("Destination timeout has elapsed, closing connection",
+              evt_tag_int("fd", log_proto_client_get_fd(self->proto)));
 
   log_pipe_notify(self->control, NC_CLOSE, self);
 }
